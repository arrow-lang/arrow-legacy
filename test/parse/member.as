name.other;
call().other;
&x.other;
