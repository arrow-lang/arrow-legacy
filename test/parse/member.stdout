<?xml version="1.0" encoding="utf-8"?>
<Module>
 <Member>
  <Identifier>name</Identifier>
  <Identifier>other</Identifier>
 </Member>
 <Member>
  <Call>
   <Expression>
    <Identifier>call</Identifier>
   </Expression>
   <Arguments/>
  </Call>
  <Identifier>other</Identifier>
 </Member>
 <AddressOf mutable="false">
  <Member>
   <Identifier>x</Identifier>
   <Identifier>other</Identifier>
  </Member>
 </AddressOf>
</Module>
